distro/otel.go

```diff
@@ -32,7 +32,13 @@ import (
 
 var errShutdown = errors.New("SDK shutdown failure")
 
-const distroVerAttr = "splunk.distro.version"
+// Deprecated
+const splunkVerAttr = "splunk.distro.version"
+
+const (
+	distroVerAttr  = "telemetry.distro.version"
+	distroNameAttr = "telemetry.distro.name"
```

Comment:
Can we reorder so that we have first name and then version (like in https://github.com/open-telemetry/semantic-conventions/blob/main/docs/resource/README.md#telemetry-distro)?